src/Public/Security.ps1

```diff
@@ -122,14 +128,40 @@ function Add-PodeSecurityHeader {
 
         [Parameter()]
         [string]
-        $Value
+        $Value,
+
+        [Parameter()]
+        [switch]
+        $Add,
```

Comment:
`-Append` might be better, `Add-PodeSecurityHeader -Add` looks strange 👀

Reply:
Yea I agree